packages/opc-menu-drawer/src/opc-menu-drawer-search.scss

```diff
@@ -0,0 +1,41 @@
+@import '@one-platform/opc-styles';
+
+button,
+::slotted(button) {
+  cursor: pointer;
+  background: none;
+  border: 0;
+}
+
+:host {
+  --opc-menu-drawer-search__bg: #efefef;
+  --opc-menu-drawer-search__margin: 0.5rem 0.5rem;
+  --opc-menu-drawer-search__padding: 0.5rem;
+  --opc-menu-drawer-search__border-radius: 8px;
+}
+
+.opc-menu-drawer-search {
+  display: flex;
+  background-color: var(--opc-menu-drawer-search__bg);
+  margin: var(--opc-menu-drawer-search__margin);
+  box-sizing: border-box;
+  align-items: center;
+  padding: var(--opc-menu-drawer-search__padding);
+  border-radius: var(--opc-menu-drawer-search__border-radius);
+
+  div {
+    padding: 0 0.25rem;
+  }
+
+  input {
+    background: none;
+    width: 100%;
+    border: none;
+    font-size: 1rem;
+    outline: none;
+  }
```

Comment:
@akhilmhdh are you sure you don't want to create a class for this? In because in future if someone else is working on this, creating any `div` inside `.opc-menu-drawer-search` would have a padding and same for `input`. If you want only direct children I suggest you to use `>` selector

Reply:
Ok that's also a good idea `>`. Yah i dont think a class is needed there as, its just a simple search with simple component

Reply:
Changed to class names as i think it will be better to understand